phpunit.xml.dist

```diff
@@ -35,6 +35,7 @@
         <server name="EXPECTED_RESPONSE_DIR" value="../Api/Responses" /> <!-- relative to tests/Application dir -->
 
         <env name="APP_ENV" value="test"/>
+        <env name="APP_DEBUG" value="0"/>
```

Comment:
It changes how errors are rendered by APIP.

Reply:
Now, if we'll have an error in tests we won't see exact information, and we have to remember to change the value here to 1. I wonder if we can handle it in a better way :thinking: